spec/system/user_spec.rb

```diff
@@ -1,18 +1,19 @@
 # frozen_string_literal: true
 
-require 'rails_helper'
+require 'spec_helper'
 
-describe User do
-  let(:user) { create :user }
+describe 'user pages' do
   let(:membership) { create :membership, admin: true}
-  let(:admin) { create :user, memberships: [membership] }
+  let!(:admin) { create :user, first_name: 'tester', memberships: [membership] }
 
   context 'viewing the index' do
     it 'directs you to the appropriate page' do
       set_current_user(admin)
       visit root_url
```

Comment:
I might suggest we use `root_path` instead. We talked about using a combination of `Rack::Test` and `Selenium` for better speed. I found in the Capybara [documentation](https://github.com/teamcapybara/capybara#gotchas) that trying to visit absolute URLs can cause issues.

Reply:
Yeah, I want to eventually do that but this was a "minimum to get things working" branch, so you can do that part.